newton/examples/sensors/example_sensor_imu.py

```
@@ -47,7 +47,7 @@ def acc_to_color(
 
 
 class Example:
-    def __init__(self, viewer):
+    def __init__(self, viewer, args):
```

Ruff (0.15.2)

[warning] Unused method argument: `args`

(ARG002)